packages/server/src/__mocks__/ioredis.ts

```diff
@@ -76,13 +76,13 @@ class Redis {
     }
   }
 
-  duplicate(): this {
-    // return new Redis(this.config);
-    return this;
+  duplicate(): Redis {
```

Comment:
Nice, I was meaning to do this in my PR but forgot, glad to see you picked it up 👍